ballerina/client.bal

```diff
@@ -221,11 +209,8 @@ public isolated client class Client {
     # + sObjectRecord - Record to be upserted
     # + return - Empty response if successful or else `error`
     @display {label: "Upsert Record"}
-    isolated remote function upsert(@display {label: "sObject Name"} string sObjectName,
-                                    @display {label: "External ID Field"} string externalIdField,
-                                    @display {label: "External ID"} string externalId,
-                                    @display {label: "Record Payload"} record {} sObjectRecord)
-                                    returns @display {label: "Result"} error? {
+    isolated remote function upsert(string sObjectName, string externalIdField, string externalId,
+            record {} sObjectRecord) returns error? {
```

Comment:
Do we need the Record postfix?
```suggestion
            record {} sObject) returns error? {
```

Reply:
updated in [6df510d](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/311/commits/6df510d1098d680d7b11812ca4ebd7b34b2af897)